src/commands/context/message/reportMessage.ts

```diff
@@ -227,6 +223,74 @@ export default contextMenuCommand(data, async ({ interaction }) => {
                 ephemeral: true,
             });
         }
+
+        const modAlertsChannel = interaction.guild.channels.resolve(constantsConfig.channels.MOD_ALERTS) as TextChannel;
+
+        if (!modAlertsChannel) {
+            await interaction.followUp({
+                content: 'Unable to find the mod alerts channel. Please contact a Moderator.',
+                ephemeral: true,
+            });
+            await scamReportLogs.send({ embeds: [reportedMessageEmbed(targetMessage, interaction, messageContent, commentContent, formattedDate)] });
+            return;
+        }
+
+        await interaction.followUp({
+            content: `Would you like to share this report in ${modAlertsChannel}? If you do not respond in 15 seconds, I will assume you do not want to share this report.`,
+            components: [
+                {
+                    type: 1,
+                    components: [
+                        {
+                            type: 2,
+                            style: 3,
+                            label: 'Yes',
+                            customId: 'shareReportYes',
+                        },
```

Comment:
I would add a No Button as well that just skips the wait. Or even if it doesn't do anything, it feels better to be able to say no.

Reply:
Yeah? I was going with less button presses if they didn't have to. I can add it in though

Reply:
I would still leave the timeout in and have no as default, but still put the button, but that's my opinion. Doesn't have to be pressed, but also doesn't suggest that you have to say yes.